library/iscsiadm_facts.py

```diff
@@ -30,7 +30,10 @@ def main():
 
         try:
             file = open('/etc/iscsi/initiatorname.iscsi', 'r')
-            text = file.readlines()[0].rstrip()
+            for line in file:
+                if line.startswith('#'):
+                    continue
+                text = line.rstrip()
```

Comment:
good job! 
I was not considering that this file could contain comments as it is usually a single line on CentOS/RHEL systems.